R/findSub.R

```diff
@@ -0,0 +1,97 @@
+# makeSeq.R

```

Comment:
Please ensure your function is properly commented for roxygen documentation

Reply:
Please also make sure that an example using your function is included so that the code is checked during Travis CI testing.